cpp/include/cudf/aggregation.hpp

```diff
@@ -286,6 +287,40 @@ std::unique_ptr<Base> make_nth_element_aggregation(
 template <typename Base = aggregation>
 std::unique_ptr<Base> make_row_number_aggregation();
 
+/**
+ * @brief Factory to create a EWMA aggregation
+ * `EWMA` returns a non-nullable column with the same type as the input,
+ * whose values are the exponentially weighted moving average of the input
+ * sequence. Let these values be known as the y_i.
+ *
+ * EWMA aggregations are parameterized by a center of mass (`com`) which
```

Comment:
Is "center of mass" standard terminology when talking about EWMAs?

Reply:
It really depends on the treatment/text. Pandas allows you to specify the same number via a bunch of different scalar transformations with different physical interpretations, but ultimately they are all funneled into one (`com`) and then the formulas are implemented in terms of that one. Since that funnel exists as an importable public pandas function, it made sense to me to roll with that one.

Reply:
Yeah now that I've finished the review I see why you chose COM, it's because of the `get_center_of_mass` method in `pandas`. I'm fine sticking with COM if there's no obvious reason to prefer another.